projects/rocprofiler-sdk/docker/Dockerfile.ci

```
@@ -0,0 +1,126 @@
+# Build a thin "base with deps" image atop the private runner image
+ARG BASE_TAG=ubuntu-22.04
+FROM rocm/rocprofiler-private:${BASE_TAG}
+
+# GPU Type from GitHub Actions
+ARG GPU_TYPE=gfx94X
+ENV GPU_TYPE=${GPU_TYPE}
+
+# Nightly Tarball Keys from GitHub Actions
+ARG GFX94X_KEY
```

```suggestion
ARG GFX94X_FILE_NAME
```

And change the rest accordingly, as the word `KEY` produces a security issue.

---

Ah yes, I was wondering why it kept yelling about that. Got confused with the s3 keys and named them KEY because of that, changed to FILE_NAME now. Thanks!